src/foam/comics/v2/DAOSummaryView.js

```diff
@@ -182,8 +182,6 @@ foam.CLASS({
       code: function() {
         if ( ! this.stack ) return;
 
-        if ( this.memento.tail )
```

Comment:
Why do you want to delete this?

Reply:
it is used when we press edit button to change memento from '....::view:id' to '.....::edit::id'.

please create a ticket if it creates a bug and I will implement "on edit memento" if a different way. thanks